azure-pipelines.yml

```diff
@@ -1,39 +1,38 @@
-trigger: none
-
-# trigger:
-#   branches:
-#     include:
-#       - main
-#   paths:
-#     include:
-#       - public
-#       - packages
-
 pool:
   name: DevCentral-Skyline
 
+trigger:
+  branches:
+    include:
+      - ni/current
+
 steps:
 - task: PowerShell@2
+  displayName: Generate version number
   inputs:
     targetType: 'inline'
     script: |
-      $ShortHash = git rev-parse --short $(Build.SourceVersion)
-      Write-Host "##vso[task.setvariable variable=hash;]$ShortHash"
+      $TagNumber = git describe --abbrev=0 --tags
+      $ImageVersion = "$TagNumber-$(Build.BuildNumber)-ni"
+      Write-Host "##vso[task.setvariable variable=ImageVersion;]$ImageVersion"
 
 - task: Docker@2
+  displayName: Build docker image
   inputs:
     repository: '963234657927.dkr.ecr.us-east-1.amazonaws.com/ni-grafana'
     command: 'build'
     Dockerfile: 'Dockerfile'
-    tags: '$(Hash)'
+    tags: '$(ImageVersion)'
 
 - task: ECRPushImage@1
+  # condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/ni/current'))
```

Comment:
This will be uncommented when this is actually ready for submission.

Reply:
It occurred to me that you could maybe stick -test on the end of the image name while testing your changes. Doesn’t seem like a big deal to have test images in ECR, and the name would make it obvious that we shouldn’t ship anything using that image. Just a thought.